buildSrc/src/main/groovy/Libraries.groovy

```diff
@@ -0,0 +1,27 @@
+import org.gradle.api.Project
+import org.gradle.api.artifacts.Dependency
+
+public class Libraries {
+    public final String junit = 'junit:junit:4.12'
```

Comment:
Yeah, good point. I wasn't too bothered of having something a little more verbose here, but I see your point. Thanks to groovy we can streamline this and still reap the benefits of using a class.